### ISSUE TITLE
Process todos from accessibility report

### ISSUE THREAD

**Issue by patcon** [GCTC-NTGC/gc-digital-talent#3099]
Mike Gifford kindly ran an accessibility report over our website and passed it to @gobyrne. Here's a hosted HTML version of it:
https://s44bv4.csb.app/

some of the issues that surface are less because of a11y concerns, but bugs in links that find a page that (rightfully) doesn't exist. We can fix them to get rid of the noise in these reports.

## To Do
- [x] process actions from the report: https://s44bv4.csb.app/ into these todos
- new apps: none!
- tc-report
  - [x] fix 2 external links ("Travailler ouvertement par défaut" section) on this page: https://talent.canada.ca/fr/projects/2000/01/01/106-standards.html
  - [x] fix 4 external links (e.g., "jobvite") on this page: https://talent.canada.ca/fr/2000/01/01/407-volume.html
  - [x] fix jekyll config (`site.baseurl`) that snuck into 2 broken link on these pages
    - https://talent.canada.ca/en/terms-and-conditions/index.html
    - https://talent.canada.ca/fr/terms-and-conditions/index.html
  - landmark issues (fixing template should fix all pages)
    - [x] "Ensures landmarks are unique" for `<nav>` element (multiple nav menus?)
    - [x] "All page content should be contained by landmarks"

## Acceptance Criteria
- [x] Changes have been made intc-report repo https://github.com/GCTC-NTGC/tc-report
- [x] Running axe browser extension on tc-report portion of site gives no errors (UPDATE: not including false positive "color contrast error"
- [ ] Code is brought into main repo via git subtree merge [(docs)](https://github.com/GCTC-NTGC/gc-digital-talent/blob/6c81c177a476395c896f12e0d7fc2b9364c472c9/tc-report/README.md)
- [ ] Bonus: Whole of current app gives no errors when axe browser extension run on it

UPDATE: PR is ready for review https://github.com/GCTC-NTGC/tc-report/pull/6

**Comment by mnigh** [comment]
@tristan-orourke can this issue be estimated?

**Comment by mnigh** [comment]
@tristan-orourke is this issue meant to be part of the current sprint?

**Comment by patcon** [comment]
The report mentions a bunch of seemingly language/locale related stuff BUT it's just an artifact of dead links, because it's rendering apache's default "page not found" (which has terrrrrible accessibility 🙃 )

> element must have a lang attribute
[WCAG 3.1.1](https://www.w3.org/TR/WCAG21/#language-of-page)

**Comment by patcon** [comment]
Related but different: https://github.com/GCTC-NTGC/gc-digital-talent/issues/3138
